File: threads/src/main/java/im/threads/internal/transport/threads_gate/responses/SendMessageData.java
package im.threads.internal.transport.threads_gate.responses;

import java.util.Date;

public class SendMessageData {
    private String messageId;
    private Date sentAt;

    public String getMessageId() {
        return messageId;
    }

    public Date getSentAt() {
        return sentAt;
    }
}

File: threads/src/main/java/im/threads/business/transport/threadsGate/responses/SendMessageData.kt
package im.threads.business.transport.threadsGate.responses;

import java.util.Date;

public class SendMessageData {
    private String messageId;
    private Date sentAt;

    public String getMessageId() {
        return messageId;
    }

    public Date getSentAt() {
        return sentAt;
    }
}
